Examples/ULC.hs

```diff
@@ -0,0 +1,12 @@
+{-# LANGUAGE PartialTypeSignatures #-}
+module Examples.ULC where
+
+import Plutarch.Core
+import Plutarch.ULC
+
+import Data.Proxy
+import Data.Functor.Identity
+import Plutarch.PType
+
+x :: forall (a :: PType) m. (IsPTypeBackend (ULCImpl m) a) => Proxy a -> ULC
```

Comment:
Why are you using IsPTypeBackend?

Reply:
thought it would fix something, but it doesn't.